source/channels/use-mattermost-google-calendar-plugin.rst

```diff
@@ -0,0 +1,71 @@
+Use the Mattermost Google Calendar plugin
+=========================================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+`The Mattermost Google Calendar plugin </about/mattermost-google-calendar-plugin.html>`__ enables you to manage events using a two-way integration between Mattermost and Google Calendar without leaving Mattermost.
+
+.. note..
+
+    Your system admin needs to `set up this plugin </about/setup-mattermost-google-calendar-plugin.html>`__ before you can use it to manage your Google Calendar in Mattermost.
+
+Connect your Google Calendar account to Mattermost
+---------------------------------------------------
+
+To use the Mattermost Google Calendar plugin, you must connect your Google Calendar account to Mattermost. You only need to complete this step once.
+
+1. Log into Mattermost using your credentials.
+2. Enter the Mattermost slash command ``/gcal connect`` in the message text field, and select **Send**.
+3. When prompted, select the link to open a new browser window and connect your Google Calendar account to Mattermost.
+4. Choose your Google account and enter your Google credentials if you're not already logged in.
+5. Select **Allow**.
+
+Mattermost will tell you when you’ve successfully connected your account. You can close the browser window and return to Mattermost.
+
+Customize your Google Calendar plugin
+---------------------------------------
+
+Mattermost prompts you to configure the plugin based on your personal preferences with the following options. You only need to complete this step once.
+
+- **Update status**: The plugin can update `your Mattermost availability </welcome/set-your-status-availability.html#set-your-availability>`__ when you have an event scheduled.
+- **Confirm status changes**: You can manually confirm every availability change, or the plugin can update your availability automatically.
+
+  - If you select **Yes**, Mattermost confirms your availability update 5 minutes before each event starts. You'll also be prompted to change your availability back to **Online** once an event ends.
+  - Select **No** to enable the plugin to update your availability automatically.
+
+- **Status during meetings**: During an event, your availability can be set to **Away** or **No Not Disturb** when you’re in a meeting. 
+
+  - Set your availability to **Away** to clearly communicate to others in Mattermost that you're unavailable. You'll continue to receive desktop, email, and push notifications based on your Mattermost notification preferences.
+  - Set your availability to `Do Not Disturb </welcome/set-your-status-availability.html#set-your-availability-as-do-not-disturb>`__  to disable all desktop, email, and push notifications.
+
+- **Receive reminders**: You can choose to receive an event reminder 5 minutes before a meeting in a direct messages, and optionally a channel message reminder if specified a channel when you created the event.
```

Comment:
I don't think channel reminders apply here, since once linked they will be sent to channels independently of the user setting. This only applies for DMs.

Reply:
Thanks! Updated in my latest commit.